src/03. Application/06. Middleware.md

```diff
@@ -187,7 +189,7 @@ Adds the middleware that resolves requests to the site root. It listens to URL's
 
 `GET /`
 
-The request is rewritten the same way as for pages.
+The startpage middleware will attempt to find a page with the slug `/home` and if found the request will be handled the same as a normal page.
```

Comment:
Hi @CrossBound! This is not completely accurate. The start page middleware executes api.Pages.GetStartpage() which will load the page with ParentId == null && SortOrder == 0, i.e the first page on root level in the site. It has nothing to do with the slug of the page.

Regards

Reply:
Interesting, in that case, the documentation here https://piranhacms.org/docs/application/routing might not be accurate. "Given that the StartpageMiddleware is registered the page with the slug /home will be resolved."

Reply:
How about we just replace that line with

"The start page middleware executes `api.Pages.GetStartpage()` which will load the page with `ParentId == null && SortOrder == 0` (i.e the first page on root level in the site). The request will then be handled the same as any other page."

Reply:
@CrossBound I see you point with the current docs referencing `/home`. I think it was meant as an example as the first section stated that there were two pages added in the structure, one with the slug `/home` and one with the slug `/about-us`. But the update text is less confusing!